### ISSUE TITLE
Cannot encode UUID based ObjectIdentifiers?

### ISSUE THREAD

**Issue by lokkilok** [PeculiarVentures/ASN1.js#64]
Perhaps (or probably) I do something wrong but I don't see how one could work with UUID based Object Identifiers, i.e. ObjectIdentifiers like 2.25.xxxx where the third part is a UUID (expressed as a 128 bit integer).

Here is some NodeJs code that tries to demonstrate the point:

```
const asn1 = require('asn1js')
const uuid = require('uuid')

function bufToBigInt (buf) { // from https://coolaj86.com/articles/convert-js-bigints-to-typedarrays/
  var hex = [];
  u8 = Uint8Array.from(buf);

  u8.forEach(function (i) {
    var h = i.toString(16);
    if (h.length % 2) { h = '0' + h; }
    hex.push(h);
  });

  return BigInt('0x' + hex.join(''));
}

function encodeOID (oid) {
  const asn1oid = new asn1.ObjectIdentifier({value: oid})
  const hexEncoded = Buffer.from(asn1oid.toBER()).toString('hex')
  console.log(`${oid} => ${hexEncoded}`)
}

// This works fine
encodeOID('2.25.1234567')


// but actual 128 bit integers don't work
const uuidAsBuffer = uuid.parse(uuid.v4())
const uuidOID = `2.25.${bufToBigInt(uuidAsBuffer)}`
encodeOID(uuidOID)
```

**Comment by lokkilok** [comment]
Made an ugly, but kind of working, fix/hack in #65

**Comment by lokkilok** [comment]
As far as I can see this is fixed in 2.2.0
